kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -22,13 +22,14 @@ define([
          *  - values = Array, the set of acceptable values
          */
         function validateSet(value, options) {
-            let errorMessage, diagnosis;
+            let errorMessage, messageId, diagnosis;
```

Comment:
Most changes here revolve around adding `messageId` to validators that didn't have them. Including `validateFloatString`.

Other changes are made to `validateFloatString` and `validateFloat` around line 330ish.